wp-content/themes/redmond/web/scss/_settings.scss

```diff
@@ -147,7 +153,7 @@ $code-padding: rem-calc(2 5 1);
 $anchor-color: $primary-color;
 $anchor-color-hover: scale-color($anchor-color, $lightness: 15%);
 $anchor-text-decoration: none;
-$anchor-text-decoration-hover: underline;
+$anchor-text-decoration-hover: none;
```

Comment:
Default should be based on user entered content in a content block which the style guide says should be underline. the menus are considered a common exception. somewhere else you can do something like `.menu a:hover { text-decoration: none; }`